content/en/docs/gke/deploy/delete-cli.md

```diff
@@ -4,14 +4,9 @@ description = "Deleting Kubeflow from GCP using the command line interface (CLI)
 weight = 6
 +++
 
-{{% alert title="Out of date" color="warning" %}}
-This guide contains outdated information pertaining to Kubeflow 1.0. This guide
-needs to be updated for Kubeflow 1.1.
-{{% /alert %}}
 
-
-This page shows you how to use the CLI to delete a Kubeflow deployment on
-Google Cloud Platform (GCP).
+This page shows you how to delete a Kubeflow deployment on
```

Comment:
Suggestion:

- Replace "shows you" with "explains"

```markdown
This page explains how to delete a Kubeflow deployment on...
```

(It's a small tweak and may look more formal though but looks more in line with the style-guides and other Google and Microsoft docs 🤷‍♂️)